Tasks/JavaToolInstallerV0/javatoolinstaller.ts

```diff
@@ -193,11 +193,21 @@ async function installPkg(pkgPath: string): Promise<string> {
 
     const newJDKs = fs.readdirSync(JDK_FOLDER).filter(jdkName => !JDKs.has(jdkName));
 
-    if (newJDKs.length !== 1) {
-        throw new Error(taskLib.loc('NewJDKIsNotInstalled'));
+    let jdkDirectory: string 
+
+    if (newJDKs.length === 0) {
+        const preInstalledJavaDirectory: string | undefined = taskLib.getVariable(extendedJavaHome);
+        if (preInstalledJavaDirectory === undefined) {
```

Comment:
Could it be instead:
```suggestion
        if (!preInstalledJavaDirectory) {
```

Reply:
Thanks! Fixed it.